### DISCUSSION TITLE
Tax rate configuration in activities instead of invoice configuration?

### DISCUSSION THREAD

**Discussion by LiaraAlis** [kimai/kimai#3479] [General]
Hi together,

I thinked about why the tax rate can be configured for invoice templates and not for the activity.
So for now it is not possible to create invoices with mixed tax rates.
I think a tax rate should be defined for a activity (and maybe a default tax rate as fallback in the invoice template) to make it possible to mix it together.
In my eyes this is also more like the reality works, isn't it?

I really don't need this feature at the moment, but I thinked about it and want to start a discussion about this topic. What do you think?

**Comment by kevinpapst** [comment]
I understand the background of that question. 
But keep the target audience of Kimai in mind: a restaurant selling drinks for 7% and food for 19% will not use Kimai.
The current logic works pretty well and is used by many companies.

**Reply by LiaraAlis** [reply]
That's what I thought. Of course, the logic is much simpler the way it is now.
I was just wondering if there are scenarios where this might be needed? 🤔

**Reply by kevinpapst** [reply]
For sure there are, but Pareto tells us to spend the time on the most used feature
